frontend/src/app/modules/main/main-page/main-page.component.html

```diff
@@ -1,6 +1,6 @@
 <div class="main-content-wrp">
-    <div class="container">
+    <div class="container-fluid">
```

Comment:
If you decide to change container class you are not longer need global boostrap variables styles